index.mjs

```diff
@@ -0,0 +1,65 @@
+//. # Warped Reducers
+//.
+//. [![Build Status](https://travis-ci.com/wearereasonablepeople/warped-reducers.svg?branch=master)](https://travis-ci.com/wearereasonablepeople/warped-reducers)
+//.
+//. Compile a standard Redux reducer from a brief definition.
+//.
+//. Works nicely with [Warped Components][1].
+//.
+//. Usage in Node depends on `--experimental-modules`.
+//. With older Node versions, use [std/esm][2].
+
+// createType :: (String, String) -> String
```

Comment:
Nice tooling for embedding md

Reply:
That's https://github.com/plaid/transcribe under the hood.